tsconfig.json

```diff
@@ -43,6 +43,9 @@
     // Test
     {
       "path": "./packages/waku/tests"
+    },
+    {
+      "path": "./examples/21_create-pages"
```

Comment:
I think there was a reason we removed example entries here (we used have them.)
@himself65 Do you remember?

Reply:
a friendly reminder on this :) - would be nice to get type checking in for these

Reply:
Yeah, I don't remember the drawback of this. Maybe just we forget adding the entry when we add a new example. (In that sense, `find` approach isn't that bad.)

Reply:
--noEmit will still emit files to `lib` or `build` for a ts monorepo

Reply:
That is unfortunate... `find` seems like a good solution to me as well then.